spec/features/user_sees_own_reviews_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'user can see all reviews left on books', %{
+  As a user,
+  I want to see all the reviews I have left on books
+  So that I can see all my reviews
+} do
+
+    # Acceptance Criteria
+    #
+    # [√] I want to see all of my reviews from my profile page
+
+  scenario 'user sees all reviews from profile page' do
+    user = FactoryGirl.create(:user)
+    author = FactoryGirl.create(:author)
+    genre = FactoryGirl.create(:genre)
+    book = FactoryGirl.create(
+      :book,
+      user_id: user.id,
+      author_id: author.id,
+      genre_id: genre.id)
+    review = FactoryGirl.create(
+      :review,
```

Comment:
Same comment as before - you can definitely refactor this into a `let` block, or even an external helper method. I would recommend doing this before you move on